src/js/components/Select/SelectContainer.js

```diff
@@ -103,7 +103,7 @@ class SelectContainer extends Component {
         onDown={this.onNextOption}
         onKeyDown={onKeyDown}
       >
-        <Box id={id ? `select-drop__${id}` : undefined} background={background}>
+        <Box id={`${id}__select-drop`} background={dropBackground}>
```

Comment:
I think we should only add an id if the user has passed one. otherwise we would have two select containers with `undefined_select-drop` ids (duplicate)